scripts/coverage_diff.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3

```

Comment:
All source files need to include a license header, for scripts you can use this format:
https://github.com/intel/fpga-runtime-for-opencl/blob/d78bdb41d2d84a42f511c14bb8fe42c1c061effe/scripts/install_aocl.sh#L1-L5
That is hash-bang, followed by license header, blank line, and then normal comments describing the file.